bugsnag-plugin-android-anr/src/main/jni/anr_google.c

```diff
@@ -39,10 +39,10 @@
 // - Register a SIGQUIT handler via sigaction()
 //
 // - Our handler callback blocks SIGQUIT (allowing the Google handler to work
-//   again) and sets a flag like "should_report_anr"
+//   again) and sets a flag like "should_report_anr_flag"
 //
-// - We have another thread waiting on the "should_report_anr" flag, which then
-//   calls bsg_google_anr_call() after a short delay to raise a SIGQUIT on
+// - We have another thread waiting on the "should_report_anr_flag" flag, which
```

Comment:
Does some of this documentation need a slight tweak to match functionality? I notice the 2 second delay no longer exists, for example.